locations/spiders/sopharmacy_bg.py

```diff
@@ -3,6 +3,8 @@
 from locations.hours import DAYS_BG, OpeningHours
 from locations.items import Feature
 
+requires_proxy = "US"  # Cloudflare bot protection used
```

Comment:
None of these were used because `requires_proxy` is needed to be a property on the spider. (Move below line 13)

Reply:
:man_facepalming: Thanks! I've completely overlooked that part.